data_diff/databases/mssql.py

```diff
@@ -155,9 +156,6 @@ def md5_as_int(self, s: str) -> str:
     def md5_as_hex(self, s: str) -> str:
         return f"HashBytes('MD5', {s})"
 
-    def normalize_uuid(self, value: str, coltype: ColType_UUID) -> str:
```

Comment:
What was a problem with this function?

Reply:
this function was a red herring, the actual issue was with limited_select